[AWS] - Fix rolearn argument bug (#25500)
demisto/content: Packs/AWS-SecurityHub/ReleaseNotes/1_2_1.md

#### Integrations

##### AWS - Security Hub

> - Fixed an issue where the *RoleArn* argument was always ignored when providing it through any command.

Reviewer: ```suggestion
- Fixed an issue in all commands that accept the *RoleArn* argument, where the *RoleArn* argument was always ignored.
```

Reviewer: And so for the rest of the RN

Author: done :)
